knowledge_graph_queries/openapi.yml

```diff
@@ -0,0 +1,133 @@
+openapi: "3.0.1"
+info:
+  title: POSTDATA knowledge graph query system
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: "0.0.1"
+servers:
+  - url: /
+    description: Production server
+  - url: http://localhost:5000/
+    description: Test server
+paths:
+  /poeticWorks:
+    get:
+      summary: List of all the poeticWorks
+      description: Gets list of all the poeticWorks
+      operationId: core.get_poeticWorks
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                example: { "head": { "vars": [ "work", "title", "creator", "date" ] }, "results": { "bindings": [ { "creator": { "type": "literal", "value": "Juana Inés de la Cruz", "xml:lang": "es" }, "title": { "type": "literal", "value": "A estos peñascos rudos", "xml:lang": "es" }, "work": { "type": "uri", "value": "http://postdata.linhd.uned.es/resource/pw_juana-ines-de-la-cruz_a-estos-penascos-rudos" } }, { "creator": { "type": "literal", "value": "Antonio Enríquez Gómez", "xml:lang": "es" }, "title": { "type": "literal", "value": "Al son de este arroyuelo, cüyo ronco", "xml:lang": "es" }, "work": { "type": "uri", "value": "http://postdata.linhd.uned.es/resource/pw_antonio-enriquez-gomez_al-son-de-este-arroyuelo-cuyo-ronco" } }, { "creator": { "type": "literal", "value": "Juana Inés de la Cruz", "xml:lang": "es" }, "title": { "type": "literal", "value": "Amado dueño mío", "xml:lang": "es" }, "work": { "type": "uri", "value": "http://postdata.linhd.uned.es/resource/pw_juana-ines-de-la-cruz_amado-dueno-mio" } }, { "creator": { "type": "literal", "value": "Juana Inés de la Cruz", "xml:lang": "es" }, "title": { "type": "literal", "value": "Amante dulce del alma", "xml:lang": "es" }, "work": { "type": "uri", "value": "http://postdata.linhd.uned.es/resource/pw_juana-ines-de-la-cruz_amante-dulce-del-alma" } } ] } }
+
+  /authors:
+    get:
+      summary: List of all the poeticWorks
+      description: Gets list of all the poeticWorks
```

Comment:
Change the description to "authors" instead of PW please

Reply:
oki